src/libopensc/reader-pcsc.c

```diff
@@ -393,7 +407,13 @@ static int refresh_attributes(sc_reader_t *reader)
 				|| rv == (LONG)SCARD_E_NO_READERS_AVAILABLE
```

Comment:
nit: on the above line, there is a typo `prevoiusly` -- not sure why this was not caught by codespell ...

Reply:
Thanks, fixed.